viewer/src/data/index.ts

```diff
@@ -13,6 +15,7 @@ export interface ButtonData {
   times?: number[];
   conversion?: { from: Reward[]; to: Reward[] };
   modal?: ModalButtonData;
+  sticky?: StickyButton;
```

Comment:
I would change to `sticky?: true | 'unstick'`

Then:

```ts
if (button.sticky === true) {
  this.stickyButton = ...;
} else if (this.button.sticky === "unstick") {
  this.stickyButton = null;
}
```

The triple equality `===` is necessary here.